app/(candidate)/dashboard/tasks/page.js

```diff
@@ -26,18 +26,13 @@ const meta = pageMetaData({
 })
 export const metadata = meta
 
-export default async function Page({ searchParams }) {
+export default async function Page() {
+  /* TODO: remove admin access once feature is released */
+  await adminAccessOnly()
+
   const campaign = await fetchCampaign()
   const tasks = await fetchTasks()
 
-  /* ------------------------------------------------------------ */
-  /* TODO: remove when feature is ready for release!!!!!!! */
-  await adminAccessOnly()
-  if (searchParams?.dt) {
-    campaign.details.electionDate = searchParams.dt
-  }
```

Comment:
You don't want to leave this in while we still are testing it and developing it behind a future flag? 🤔